StereoKit/Native/NativeEnums.cs

```diff
@@ -5,740 +5,766 @@
 using System;
```

Comment:
This file is actually generated automatically by parsing `stereokit.h`! If you run the StereoKitAPIGen project in the solution, you'll see this file update automatically :)

Reply:
Since this file wasn't changed, could you restore it to its initial state? Running StereoKitAPIGen  should do this quickly.